CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
-0.77.x (TBD)
+0.78.0 (TBD)
```

Comment:
My thinking is that the person responsible for adding an entry in changelog is making an assertion that the entry is sufficiently different to bump the version. Sorting out a precise minor number would be done in the release branch if necessary.